community/ibm-geode-dev/README.md

```diff
@@ -0,0 +1,114 @@
+
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Introduction
+
+This chart bootstraps a [Geode](https://hub.docker.com/r/ppc64le/geode ) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/ibm-geode-dev
+```
+
+## Default Credentials
+The default credentials for the Geode service are username - admin and password - admin
+
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Chart Details
+This chart bootstraps a [Geode](https://hub.docker.com/r/ppc64le/geode ) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+
+## Configuration
+
+The following table lists the configurable parameters of the Geode chart and their default values.
+
+|      Parameter            |          Description            |                         Default                         |
+|---------------------------|---------------------------------|---------------------------------------------------------|
+| `image`                   | The image to pull and run       | ppc64le/geode:v1.2.1                                    |
+| `imagePullPolicy`         | Image pull policy               | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
+| `nodeSelector`            | Specify what architecture Node  | `ppc64le`                                               |
+
+
+The above parameters map to `ibm-geode-dev` params.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. 
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml stable/ibm-geode-dev
+```
+
+> **Tip**: You can use the default `values.yaml`
+
+## Support
+
+The helm charts are provided "as-is" and without warranty of any kind.
+
+All helm charts and packages are supported through standard open source forums and helm charts are updated on a best effort basis.
+
+Any issues found can be reported through the links below, and fixes may be proposed/submitted using standard git issues as noted below.
+
+[Submit issue to Helm Chart] ( https://github.com/ppc64le/charts/issues )
+
+[Submit issue to Geode docker image]  ( https://github.com/ppc64le/build-scripts/issues )
+
+[Submit issue to Geode open source community] ( https://issues.apache.org/jira/projects/GEODE/issues/GEODE-6152?filter=allopenissues )
+
+
+
+## Note (Cluster Image Security)
```

Comment:
Remove this whole section; `docker.io/ibmcom/*` is already in the default ICP cluster-wide whitelist